Skype/WebSDK/docs/PTAudioIncoming.md

```diff
@@ -0,0 +1,91 @@
+
+# Incoming P2P/PSTN Call
+
+ _**Applies to:** Skype for Business 2015_
+
+## Listening for incoming call notifications
+
+When a remote user starts a call we will receive an invitation to join the call.
+In order to see the notification we need to:
+1. Listen to the conversation collection for newly added conversations
+ ```js
+application.conversationsManager.conversations.added(function (conversation) {
+    // ...
+});
+```
+2. For every added conversation we need to observer the `audioService.accept.enabled` command.
+```js
+conversation.audioService.accept.enabled.when(true, function () {
+    // ....
+})
+```
+3. When the command becomes available we have received a notification. We can now prompt the user to accept ot decline the invitation.
+When the user accepts, we execute `conversation.audioService.accept()`. When they reject `conversation.audioService.reject()` is executed.
+```js
+if (confirm('Accept incoming Audio invitation?')) {
+    conversation.audioService.accept();
+} else {
+    conversation.audioService.reject();
+}
+```
+
+## Conversation Call State
+We can subscribe to the conversation call state to get information about the overall call status.
+For example: Is there an ongoing call in this conversation. This does not mean that we are connected to the call.
+
+```js
+conversation.state.changed(function (newValue, reason, oldValue) {
+    //...
+});
+```
+
+**Possible call states:**
```

Comment:
Change to "possible conversation states" - we want to minimize confusion between conversation states and the states of individual modalities, and "call" is an ambiguous word.